example/android/app/src/main/res/values/dark_theme.xml

```diff
@@ -0,0 +1,245 @@
+        <item name="windowActionModeOverlay">true</item>
+
+        <item name="alertDialogTheme">@style/PSPDFCatalog.AlertDialog.Dark</item>
+        <item name="android:listDivider">@android:drawable/divider_horizontal_dark</item>
+
+        <!-- Default background color of PSPDFKit Views. PDF view background is controlled by a separate property pspdf__backgroundColor -->
+        <item name="android:colorBackground">@color/dark_background_color</item>
+        <!-- Default foreground color of PSPDFKit Views. -->
+        <item name="android:colorForeground">@color/dark_input_field_blue</item>
+
+        <item name="pspdf__contextualToolbarStyle">@style/PSPDFCatalog.ContextualToolbar.Dark</item>
+        <item name="pspdf__actionBarIconsStyle">@style/PSPDFCatalog.ActionBarIcons.Dark</item>
+
+        <item name="pspdf__backgroundColor">@color/dark_background_color</item>
+        <item name="pspdf__navigationHistoryIconColor">@color/dark_accent</item>
+        <item name="pspdf__inlineSearchStyle">@style/PSPDFCatalog.SearchViewInline.Dark</item>
+        <item name="pspdf__modularSearchStyle">@style/PSPDFCatalog.SearchViewModular.Dark</item>
+        <item name="pspdf__thumbnailBarStyle">@style/PSPDFCatalog.ThumbnailBar.Dark</item>
+        <item name="pspdf__outlineViewStyle">@style/PSPDFCatalog.OutlineView.Dark</item>
+        <item name="pspdf__thumbnailGridStyle">@style/PSPDFCatalog.ThumbnailGrid.Dark</item>
+        <item name="pspdf__toolbarCoordinatorLayoutStyle">@style/PSPDFCatalog.ToolbarCoordinator.Dark</item>
+        <item name="pspdf__searchResultHighlighterStyle">@style/PSPDFCatalog.SearchResultHighlighter.Dark</item>
+        <item name="pspdf__annotationSelectionStyle">@style/PSPDFCatalog.AnnotationSelection.Dark</item>
+        <item name="pspdf__textSelectionStyle">@style/PSPDFCatalog.TextSelection.Dark</item>
+        <item name="pspdf__propertyInspectorStyle">@style/PSPDFCatalog.PropertyInspector.Dark</item>
+        <item name="pspdf__actionMenuStyle">@style/PSPDFCatalog.ActionMenu.Dark</item>
+        <item name="pspdf__stampPickerStyle">@style/PSPDFCatalog.StampPicker.Dark</item>
+        <item name="pspdf__signatureLayoutStyle">@style/PSPDFCatalog.SignatureLayout.Dark</item>
+        <item name="pspdf__documentViewStyle">@style/PSPDFCatalog.DocumentViewStyle.Dark</item>
+        <item name="pspdf__sharingDialogStyle">@style/PSPDFCatalog.SharingDialog.Dark</item>
+        <item name="pspdf__newPageDialogStyle">@style/PSPDFCatalog.NewPageDialog.Dark</item>
+        <item name="pspdf__modalDialogStyle">@style/PSPDFCatalog.ModalDialog.Dark</item>
+        <item name="pspdf__formSelectionStyle">@style/PSPDFCatalog.FormSelection.Dark</item>
+        <item name="pspdf__annotationNoteHinterStyle">@style/PSPDFCatalog.AnnotationNoteHinter.Dark</item>
+    </style>
+
+    <!-- Action bar icon items. -->
+    <style name="PSPDFCatalog.ActionBarIcons.Dark" parent="PSPDFKit.ActionBarIcons">
```

Comment:
These theme styles would require renaming too. They should not mention the catalog either.